.github/ISSUE_TEMPLATE/bump_libraries.md

```diff
@@ -17,6 +17,18 @@ This script can be used to do most of the work:
 
 https://github.com/gazebo-tooling/release-tools/blob/master/release-repo-scripts/bump_dependency.bash
```

Comment:
I was checking what the args are and noticed that the path of this file incorrect: 

```suggestion
https://github.com/gazebo-tooling/release-tools/blob/master/source-repo-scripts/bump_dependency.bash
```

Reply:
wow, I don't know where that came from